src/node_options.h

```diff
@@ -0,0 +1,60 @@
+#ifndef SRC_NODE_OPTIONS_H_
+#define SRC_NODE_OPTIONS_H_
+
+#include "node_version.h"
+#include "util.h"
+#include "getopt.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+namespace node {
+
+class NodeOptions {
+ public:
+  bool print_eval = false;
+  bool force_repl = false;
+  bool trace_deprecation = false;
+  bool throw_deprecation = false;
+  bool abort_on_uncaught_exception = false;
+  bool trace_sync_io = false;
+  const char* eval_string = nullptr;
+  unsigned int preload_module_count = 0;
+  const char** preload_modules = nullptr;
+  bool use_debug_agent = false;
+  bool debug_wait_connect = false;
+  int debug_port = 5858;
+  bool no_deprecation = false;
+#if defined(NODE_HAVE_I18N_SUPPORT)
+  // Path to ICU data (for i18n / Intl)
+  const char* icu_data_dir = nullptr;
+#endif
+
+  NodeOptions();
+  ~NodeOptions();
```

Comment:
You can drop the constructor and destructor if they're just no-ops.  Can you make the class non-copyable by adding `DISALLOW_COPY_AND_ASSIGN(NodeOptions);` at the bottom?